src/walletConnect/request.ts

```diff
@@ -22,13 +23,34 @@ import Web3 from 'web3'
 
 const TAG = 'WalletConnect/handle-request'
 
-export function* handleRequest({ method, params }: { method: string; params: any[] }) {
-  const wallet: ViemWallet = yield* call(getViemWallet, networkConfig.viemChain.celo)
+export const networkIdToWalletConnectChainId: Record<NetworkId, string> = {
```

Comment:
i opted for these mappings because the chainId for walletconnect contains the namespace so we cannot use the chainId directly. we would need to make assumptions about the format of the chainId (i.e. split by `:` and take the second element) which seems just as brittle as having a custom mapping...

Reply:
yes good idea. i moved it to networkConfig because of the second point...although i was tempted to put it in the wc/types.ts file

Reply:
my naive guess is if we add in the future another EVM compatible network like Polygon, probably it would be simpler to copy-paste and fill some _template_ rather than sniffing all around the codebase to find all the places to insert it